test/api.js

```diff
@@ -42,7 +42,11 @@ module.exports = function api(smocks) {
     label: 'App stats',
     path: `${BASE_URL}/apps/{guid}/stats`,
     handler: function (req, reply) {
-      reply(appStats);
+      var app = apps.filter((a) => a.guid === req.params.guid).pop();
+      var response = Object.assign({}, appStats, {
+        state: app.state
```

Comment:
Should we also set current mem and disk to 0 or null so it's a more correct response?